.env.example

```diff
@@ -1,4 +1,6 @@
-NODE_LOCAL_PORT=4030
+PORT=3000
+NODE_LOCAL_PORT=3000
```

Comment:
is this still getting used?

Reply:
no its not, heroku automatically assign an available PORT while running, i added these just in case something happens and it could not load the automatic $PORT in the production so it fetch from the manually added env in heroku dashboard